sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateExec.scala

```diff
@@ -425,7 +432,10 @@ case class TransformWithStateExec(
 
     val operatorPropertiesJson: JValue =
       ("timeMode" -> JString(timeMode.toString)) ~
-        ("outputMode" -> JString(outputMode.toString))
+        ("outputMode" -> JString(outputMode.toString)) ~
```

Comment:
Can we just move this to a class as well ? Something like `case class TransformWithStateOperatorProperties` and apply the similar to/from json methods ?